pkg/core/test_util/random_util.go

```diff
@@ -9,29 +9,29 @@ import (
 )
 
 // RandomString returns a random string with the n as its length.
-func randomString(n int) string {
+func RandomString(n int) string {
 	b := make([]byte, n)
 	for i := range b {
-		b[i] = byte(randomInt(65, 90))
+		b[i] = byte(RandomInt(65, 90))
 	}
 
 	return string(b)
 }
 
 // RandomInt returns a random integer between min and max.
-func randomInt(min, max int) int {
+func RandomInt(min, max int) int {
```

Comment:
We should mention that `max` is non-inclusive: `integer in [min,max)`

Reply:
sure, I will